### DISCUSSION TITLE
css/topics/position-fixed

### DISCUSSION THREAD

**Discussion by giscus** [wind-stone/blog#317] [Announcements]
# css/topics/position-fixed

position: fixed 失效 若子孙元素是position: fixed，则当其某一祖先元素创建了层叠上下文（Stacking Context，也称为堆叠上下文），则该子孙元素的固定定位将基于该祖先元素进行定位，而不是基于 Viewport，因此固定定位失效了。 参考MDN - 层叠上下文，可以看到其中有很多方式能够让元素创建层叠上下文，但不是...

https://blog.windstone.cc/css/topics/position-fixed.html

**Comment by jiechliu** [comment]
6
